app/proguard-rules.pro

```diff
@@ -1,5 +1,128 @@
 -dontobfuscate
 
+####################################################################################################
+# Copy of `proguard-android-optimize.txt`.
+# We need to _remove_ some of the configuration, so the only way to achieve that is to dump it here
+# and modify it.
```

Comment:
Let's be explicit and add some info about:
- the modifications we made, what issue number they were made in, and (in this case) link to the R8 bug so we know when this can be removed
- the version of this file we're using (e.g. file name for me locally is proguard-android-optimize.txt-3.5.0)